src/rollback_to_stable/rts_api.c

```diff
@@ -115,6 +115,10 @@ __rollback_to_stable_int(WT_SESSION_IMPL *session, bool no_ckpt)
     WT_ASSERT_SPINLOCK_OWNED(session, &conn->checkpoint_lock);
     WT_ASSERT_SPINLOCK_OWNED(session, &conn->schema_lock);
 
+    /* Check this nice and early, before we do any prep work. */
+    if (__wt_conn_is_disagg(session))
```

Comment:
Could you add a log message here, just so that it gets flagged (probably at a higher than default log level)

Reply:
Done - see https://github.com/wiredtiger/wiredtiger/pull/11413#issuecomment-2588896404.